geth/client.go

```diff
@@ -491,6 +494,39 @@ func (c *Connector) SendTransaction(rawTx []byte) error {
 	return nil
 }
 
+// ConfirmedBalance returns number of funds available under control of
+// connector.
+//
+// NOTE: Part of the common.Connector interface.
+func (c *Connector) ConfirmedBalance(account string) (string, error) {
+	m := crypto.NewMetric(c.cfg.DaemonCfg.Name, string(c.cfg.Asset),
+		MethodConfirmedBalance, c.cfg.Metrics)
+	defer m.Finish()
+
+	balance := "0"
+
+	err := c.db.Update(func(tx *bolt.Tx) error {
+		accountsBucket, err := tx.CreateBucketIfNotExists(accountsToAddressesBucket)
+		if err != nil {
+			m.AddError(errToSeverity(ErrDatabase))
+			return errors.Errorf("unable to get accounts bucket: %v", err)
+		}
+		addressBytes := accountsBucket.Get([]byte(account))
+		if addressBytes == nil {
+			return nil
+		}
+		weis, err := c.client.EthGetBalance(string(addressBytes), "latest")
+		if err != nil {
+			return err
+		}
+		ethers := float64(weis.Uint64()) / math.Pow10(18)
+		balance = strconv.FormatFloat(ethers, 'f', -1, 64)
```

Comment:
better use decimals

Reply:
Yes, you are right. But i just make it similar to other *Balance methods.